docs/how-to-maintain.rst

```diff
@@ -0,0 +1,97 @@
+How To Maintain
+===============
+
+This document is dedicated to maintainers of GETTSIM.
+
+
+Versioning
+----------
+
+GETTSIM adheres in large parts to `semantic versioning <https://semver.org>`_. Thus, for
+a given version number ``major.minor.patch``
+
+* ``major`` is incremented when you make incompatible API changes.
+* ``minor`` is incremented when you add functionality which is backwards compatible.
+* ``patch`` is incremented when you make backwards compatible bug fixes.
+
+Branching Model
+---------------
+
+The branching model for GETTSIM is very simple.
+
+1. The master branch is the target of ongoing development. Every PR should be merged
+   into master.
+
+   (The only deviation can happen with respect to backports. See :ref:`backports` for
+   more.)
+
+2. There can exist branches for minor versions called, for example, ``0.1.``. The
```

Comment:
Minor versions have their own branches. Bugfixes are included in these branches and patch releases are made from these branches.

Once a minor version is not supported anymore (=will not receive any more bugfixes), the branch should be deleted for clarity.

Reply:
Done.